features/bevragen/zoek-met-nummeraanduiding-identificatie/overzicht.feature

```diff
@@ -0,0 +1,111 @@
+    | 0599                              |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | gemeente_code | identificatiecode nummeraanduiding (11.90) |
+    | 0599          | 0599200000219679                           |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                          | waarde                               |
+    | type                          | ZoekMetNummeraanduidingIdentificatie |
+    | nummeraanduidingIdentificatie | 0599200000219679                     |
+    | fields                        | burgerservicenummer                  |
+    Dan heeft de response 1 persoon
+    En heeft de response een persoon met alleen de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 000000024 |
+
+  Scenario: Er zijn geen personen ingeschreven op het adres met de opgegeven nummeraanduiding identificatie
+    Als personen wordt gezocht met de volgende parameters
+    | naam                          | waarde                               |
+    | type                          | ZoekMetNummeraanduidingIdentificatie |
+    | nummeraanduidingIdentificatie | 0518200000366054                     |
+    | fields                        | burgerservicenummer                  |
+    Dan heeft de response 0 personen
+
+  Scenario: Er zijn meerdere personen ingeschreven op het adres met de opgegeven nummeraanduiding identificatie
+    Gegeven een adres heeft de volgende gegevens
+    | gemeente_code | identificatiecode nummeraanduiding (11.90) |
+    | 0518          | 0518200000617227                           |
+    En de persoon met burgerservicenummer '000000024' is ingeschreven op het adres met de volgende gegevens
+    | gemeente van inschrijving (09.10) |
+    | 0518                              |
+    En de persoon met burgerservicenummer '000000025' is ingeschreven op het adres met de volgende gegevens
+    | gemeente van inschrijving (09.10) |
+    | 0518                              |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                          | waarde                               |
+    | type                          | ZoekMetNummeraanduidingIdentificatie |
+    | nummeraanduidingIdentificatie | 0518200000617227                     |
+    | fields                        | burgerservicenummer                  |
+    Dan heeft de response 2 personen
+    En heeft de response een persoon met alleen de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 000000024 |
+    En heeft de response een persoon met alleen de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 000000025 |
+
+Rule: De optionele 'inclusiefOverledenPersonen' parameter moet worden opgegeven om een overleden persoon te kunnen vinden
+
+  Abstract Scenario: Zoek overleden personen die zijn ingeschreven op het adres met de opgegeven nummeraanduiding identificatie
```

Comment:
Vraagje: Blijven een overleden personen altijd ingeschreven op het laatste adres waarop deze ingeschreven waren?

Zo niet, dan is onderstaande request erg afhankelijk van het moment dat overleden personen uitgeschreven worden op een adres. Het ene moment vind je deze wel het andere moment niet meer.

Reply:
Inmiddels is duidelijk dat overleden personen inderdaad ingeschreven blijven op het laatste adres waarop deze ingeschreven waren.